src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -381,7 +382,7 @@ std::vector<index_t> subtract_sets(const std::vector<index_t> &v1,
             res.push_back(v1[i1]);
         }
     }
-    return res;
+    return std::vector<index_t>(std::move(res));
 }
```

Comment:
looks good, now that we are c++11 we do this. 

(I will still probably be a dinosaur and use out refs, b/c they are a known qty)